src/System.Private.CoreLib/shared/System/TimeSpan.cs

```diff
@@ -397,11 +394,11 @@ public static bool TryParseExact(string input, string format, IFormatProvider fo
             return TimeSpanParse.TryParseExact(input, format, formatProvider, TimeSpanStyles.None, out result);
         }
 
-        public static bool TryParseExact(ReadOnlySpan<char> input, ReadOnlySpan<char> format, IFormatProvider formatProvider, out TimeSpan result)
+        public static bool TryParseExact(ReadOnlySpan<char> input, ReadOnlySpan<char> format, IFormatProvider? formatProvider, out TimeSpan result)
         {
             return TimeSpanParse.TryParseExact(input, format, formatProvider, TimeSpanStyles.None, out result);
         }
-        public static bool TryParseExact(string input, string[] formats, IFormatProvider formatProvider, out TimeSpan result)
+        public static bool TryParseExact(string? input, string?[]? formats, IFormatProvider? formatProvider, out TimeSpan result)
```

Comment:
I wonder why the compiler didn't warn, but formats can't be null... https://source.dot.net/#System.Private.CoreLib/shared/System/Globalization/TimeSpanParse.cs,1675

Reply:
Formats can be null; it'll just return false in that case.